front_end/webserver.py

```diff
@@ -388,6 +389,23 @@ def post(self, user_id):
         except Exception as inst:
             render_error(self, traceback.format_exc())
 
+class UnregisterHandler(BaseUserHandler):
+    def post(self, course, user_id):
+        try:
+            role = self.get_current_role()
```

Comment:
This function no longer exists. Please look at other handlers for examples of how to check whether the user is an admin, etc. And update this accordingly.